src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -275,6 +275,80 @@ namespace OpenLoco::Ui::PromptBrowse
         *scrollHeight = window->row_height * _numFiles;
     }
 
+    // 0x004464F7
+    static void onScrollMouseDown(window* self, int16_t x, int16_t y, uint8_t scrollIndex)
+    {
+        auto index = y / self->row_height;
+        if (index > _numFiles)
+            return;
+
+        Audio::playSound(Audio::sound_id::click_down, self->x + (self->width / 2));
+
+        file_entry entry = _files[index];
+
+        // Clicking a directory, with left mouse button?
+        if (Input::state() == Input::input_state::scroll_left && entry.is_directory())
```

Comment:
We use this check throughout, but I think this enum is either confusingly named, or wrong. This state appears to be just a normal left click, at any rate.

Reply:
Its the state for a left click on a scroll area.